### DISCUSSION TITLE
@next/font flashing when third party font is injected

### DISCUSSION THREAD

**Discussion by beamercola** [vercel/next.js#43862] [Help]
When using a third party banner widget (Wisepops) that annoyingly injects a google font, it causes my @next/font to flash.  Initially I assumed it was from the default `display: optional` but I can't seem to find a way to fix this

Example: 
https://k8qenr.sse.codesandbox.io/

Sandbox (Script doesn't work in the iframe - must use the pop-out preview):
https://codesandbox.io/s/wisepops-next-13-font-k8qenr?file=/pages/_app.tsx

**Comment by ciruz** [comment]
FYI its not happening in Safari.
